net-misc/ytdl-gui/ytdl_gui-1.0.recipe

```diff
@@ -0,0 +1,79 @@
+SUMMARY="A simple-to-use, cross-platform graphical interface for youtube-dl"
+DESCRIPTION="YouTubeDL-GUI is a cross-platform graphical interface for \
+youtube-dl, a downloader for online media sharing services like YouTube,\
+ Vimeo, SoundCloud, Bandcamp, and more. It is based on Qt and aims to be\
```

Comment:
To be nice and consistent, the spaces should be moved from the start of the line to the end of the previous line. As it's done already for the above lines.